.travis.yml

```diff
@@ -38,10 +25,42 @@ before_install:
 install:
   - yarn install --no-lockfile --non-interactive
```

Comment:
If we're doing a `no-lockfile` install before all our tests run, isn't the default `test` stage already a "floating dependencies" pass?

If I'm understanding the intent right, I think we could probably just remove this global `install` step and let Travis do its default thing (which would be a regular `yarn install`)